crates/bevy_winit/src/winit_windows.rs

```diff
@@ -63,18 +63,20 @@ impl WinitWindows {
                     window_descriptor.height as u32,
                 )),
             )),
-            _ => {
+            WindowMode::Windowed => {
                 if let Some(sf) = scale_factor_override {
                     winit_window_builder.with_inner_size(logical_size.to_physical::<f64>(sf))
                 } else {
                     winit_window_builder.with_inner_size(logical_size)
                 }
             }
+        };
+
+        winit_window_builder = winit_window_builder
```

Comment:
I guess it's not an issue, but it seems to me like most of those settings simply won't do anything when not in windowed mode

Reply:
`always_on_top` and `transparent` work in all modes.
The `resizable` and `decorations` settings will still take effect if the window is switched to windowed mode later on.

Reply:
Ah, right, didn't think about switching mode, makes sense.